templates/adherent/_profile_menu.html.twig

```diff
@@ -6,6 +6,10 @@
             <a href="{{ path('app_user_profile') }}">Mon compte</a>
         </li>
 
+        <li class="{{ route == 'app_enmarche_user_profiledonation' ? 'active' }}">
+            <a href="{{ path('app_enmarche_user_profiledonation') }}">Mes dons</a>
```

Comment:
such route does not exist (yet)

Reply:
no it exist

Reply:
done

Reply:
yeah my bad. i m not used to route name autoconfiguration on this project :)

Reply:
:'(